src/CommunitiesListItem.cpp

```diff
@@ -98,11 +113,33 @@ CommunitiesListItem::resolveName() const
 {
         if (!name_.isEmpty())
                 return name_;
-
+        if (groupId_.startsWith("tag:"))
+                return groupId_.right(groupId_.size() - 4);
         if (!groupId_.startsWith("+"))
                 return QString("Group"); // Group with no name or id.
 
         // Extract the localpart of the group.
         auto firstPart = groupId_.split(':').at(0);
         return firstPart.right(firstPart.size() - 1);
 }
+
+void
+CommunitiesListItem::updateTooltip()
+{
+        if (groupId_ == "world")
+                setToolTip(tr("All rooms"));
+        else if (is_tag()) {
+                QString tag = groupId_.right(groupId_.size() - 4);
```

Comment:
It would be nice to have this `4` number derived from the `tag:` string somehow. Currently it seems like a magic number if you only read a part of the code.

Reply:
Is adding a comment explaining it enough, or should I replace it by `strlen("tag:")`?

Reply:
I've seen it in 2-3 places so maybe it would be better to use something like `strlen`.